python/tank/descriptor/io_descriptor/rez.py

```diff
@@ -93,7 +93,7 @@ def _get_rez_pkg_location(self):
 
         path = stdout.strip()
         log.debug("Resulting path is: {0}".format(path))
-        return path
+        return path.decode('utf-8')
```

Comment:
I'm not sure if this will work properly with Python 2. We might want to do something like:

```python
if sys.version_info[0] == 2:
    return path
else:
    return path.decode('utf-8')
```

Also, I'm not sure the encoding is always `utf-8` or if it is the `locale.getdefaultlocale()`.

Reply:
ajust the code by using the `locale.getdefaultlocale()`